phpunit.xml.dist

```diff
@@ -301,11 +301,12 @@
 	</testsuite>
 	<testsuite name="paratest-sox-snowflake">
 		<!--This is temporary to check something-->
-		<directory>tests/Backend/SOX</directory>
+		<file>tests/Backend/SOX/SOXTokensTest.php</file>
 	</testsuite>
 	<testsuite name="sync-sox-snowflake">
 		<!--This is temporary to check something-->
 		<file>tests/Backend/SOX/SOXCommonTest.php</file>
+		<file>tests/Backend/SOX/MergeRequestsTest.php</file>
```

Comment:
toto je TMP tady to zas opravím https://github.com/keboola/storage-api-php-client/pull/1107